packages/payload/src/admin/components/views/Dashboard/index.tsx

```diff
@@ -29,6 +34,12 @@ const Dashboard: React.FC = () => {
     setStepNav([])
   }, [setStepNav])
 
+  useEffect(() => {
+    if (actions) {
```

Comment:
It seems we would always want to call setViewActions([]) for dashboard. Otherwise if you're navigating from another view that has actions it won't clear the actions that were present on the old view.